commonDependencyVersionConstraints/build.gradle

```
@@ -101,6 +101,7 @@ dependencies {
 
     // make sure that this is compatible with the testcontainers version
     api group: 'org.opensearch', name: 'opensearch-testcontainers', version: '2.0.1'
+    api group: 'org.testcontainers', name: 'toxiproxy', version: '1.19.8'
```

This should be clustered with the other `org.testcontainers` resources and reusing the variable
```suggestion
    api group: 'org.testcontainers', name: 'toxiproxy', version: testcontainers
```